hyp3_testing/compare.py

```diff
@@ -124,6 +124,17 @@ def compare_cf_spatial_reference(reference: xr.Dataset, secondary: xr.Dataset):
         )
 
 
+def compare_raster_info(reference: Path, secondary: Path):
```

Comment:
This function deserves testing if we're going to keep it.  I still want to explore `rioxarray` + the existing `compare_cf_spatial_reference()` for doing this check.

Reply:
`rioxarray` won't solve this problem unfortunately because it doesn't specify a global `Conventions` attribute:

```python
rio_rx = rioxarray.open_rasterio(reference)
rio_rx
<xarray.DataArray (band: 1, y: 4591, x: 8888)>
[40804808 values with dtype=float32]
Coordinates:
  * band         (band) int64 1
  * y            (y) float64 5.922e+06 5.922e+06 ... 5.784e+06 5.784e+06
  * x            (x) float64 3.946e+05 3.946e+05 ... 6.612e+05 6.612e+05
    spatial_ref  int64 0
Attributes:
    _FillValue:    0.0
    scale_factor:  1.0
    add_offset:    0.0
    grid_mapping:  spatial_ref
```